data/countries.json

```diff
@@ -2810,12 +2816,23 @@
   {
     "alpha2": "AX",
     "alpha3": "ALA",
-    "independent": "Part of FI",
+    "independent": "No",
     "name": {
       "short": "Åland Islands",
       "uppercase": "ÅLAND ISLANDS"
     },
     "numeric": "248",
     "status": "officially-assigned"
+  },
+  {
```

Comment:
@ruimarinho considering only countries with status `officially-assigned` and `transitionally-reserved`this is missing in current version.